src/ListItem/ListItem.react.js

```diff
@@ -268,9 +269,8 @@ class ListItem extends PureComponent {
                 secondaryText = centerElement.secondaryText;
                 tertiaryText = centerElement.tertiaryText;
             }
-            const numberOfLines = !tertiaryText ?
-                getNumberOfSecondaryTextLines(this.state.numberOfLines) : 1;
-
+            const secondLineNumber = typeof tertiaryText === 'undefined' ? numberOfLines : 1;
```

Comment:
I see what you're saying. I think that changing `type of tertiaryText` to `!tertiaryText` might work better too, let me experiment.

Line 272 is basically saying: if there isn't a tertiaryText prop then the numberOfLines prop for the Text will be the numberOfLines const on 255, but if there is a tertiaryText prop then the second line should only ever be 1 line.

Similarly, line 273 does that same thing. I attached a picture with 4 different examples of cards I made with both two and three lines of text/. This is the code for it:

```
          <Card flex={1}>
            <ListItem
              leftElement={<InitialsAvatar name={'Matthew Benjamin'} color={'red'} />}
              centerElement={{primaryText: 'Primary Text', secondaryText: 'Secondary Text'}}
              numberOfLines={2}
            />
          </Card>
          
          <Card flex={1}>
            <ListItem
              leftElement={<InitialsAvatar name={'Matthew Benjamin'} color={'red'} />}
              centerElement={{primaryText: 'Primary Text', secondaryText: 'This is long text to show off how the dynamic text field works with only two lines!'}}
              numberOfLines='dynamic'
            />
          </Card>
          
          <Card flex={1}>
            <ListItem
              leftElement={<InitialsAvatar name={'Matthew Benjamin'} color={'red'} />}
              centerElement={{primaryText: 'Primary Text', secondaryText: 'Secondary Text', tertiaryText: 'Tertiary Text'}}
              numberOfLines={3}
            />
          </Card>

        <Card flex={1}>
            <ListItem
              leftElement={<InitialsAvatar name={'Matthew Benjamin'} color={'red'} />}
              centerElement={{primaryText: 'Primary Text', secondaryText: 'Secondary Text', tertiaryText: 'This is long text to show off how the dynamic text field works with only three lines!'}}
              numberOfLines='dynamic'
            />
          </Card>
```
![image](https://cloud.githubusercontent.com/assets/9419421/22598782/aae1f63e-ea02-11e6-9dee-52cfcfd06974.png)